core/src/main/scala/chisel3/internal/firrtl/Converter.scala

```diff
@@ -363,6 +363,14 @@ private[chisel3] object Converter {
         id.desiredName,
         params.keys.toList.sorted.map { name => convert(name, params(name)) }
       )
+    case ctx @ DefIntModule(id, name, ports, topDir, params) =>
```

Comment:
i am less concerned about the use of `Int` on the internal (another overloading of the abbreviation!) interface, but for the ones we give to users please let's call them `Intrinsic`